src/content/1.7/development/components/grid/tutorials/work-with-grid-actions.md

```diff
@@ -78,7 +78,7 @@ final class CustomerGridDefinitionFactory extends AbstractGridDefinitionFactory
 As you see, Grid stores Grid actions using a `GridActionCollection` object. This means that every action needed for Grid must be added to `GridActionCollection` using `add()` method.
 You'll find the list of existing actions in the documentation.
 
-## How to add Bulk actions to a Grid using a module?
+## How to add Grid actions to a Grid using a module?
```

Comment:
Grid seems redundant
```suggestion
## How to add actions to a Grid using a module?
```

Reply:
I agree but since we already have Bulk actions and Row actions, it's good to have a way to designate these actions And it matches the classes used